docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
+- Projects do not have a start or end date (start or end dates are specified on allocations, users and attributions)
+- New projects must have a name, description, project owner and institution
+ 
+### Project administration
+- Principal investigators can associate new/existing users to an existing projects by email address
+- Associating users to a project should fail after a specified number of user has been reached (to avoid excessive disk space to a project)
+- The associated user limit should be set on a per-project basis and have a per-institution limit
+- Users associated with a project should be required to click a link in a confirm email before being added
```

Comment:
Users have to apply to join, we can add an email notification for when the Technical Lead authorises them.

Reply:
Added to project board.